jest.config.js

```diff
@@ -11,6 +11,12 @@ module.exports = {
     '@stencil/core/testing': '<rootDir>/testing/index.js',
     '@utils': '<rootDir>/src/utils',
   },
+  coverageDirectory: './test-results/',
+  coverageReporters: ["json", "lcov", "text", "clover", "json-summary"],
+  collectCoverageFrom: [
```

Comment:
For now, only `cli` and `utils` are collected for coverage. Without these lines, the `jest --coverage` command will take quite a while.

Reply:
Consider opening this up to all directories. IMO this doesn't provide a ton of value if we're trying to make this a team-wide initiative (if someone were to open a PR without any tests in any other directory we're back to square one). Rather consider creating either a new script in package.json for coverage and/or documenting that coverage can be determined by running `npm run test -- --coverage` (that double dash is important, it helps pipe `--coverage` to the `jest` command)

Reply:
The tests aren't running as a part of the build process though 🤔 Did you mean CI time went up? Can you share the configuration you were using?

Reply:
Yeah! Just delete this line and run the test command. You'll see what I mean. There's an initial compilation process that Jest/Istanbul use to keep track of lines covered during the tests.

Reply:
I wonder if there's a configuration error with deleting `collectCoverageFrom`, as 
```
  collectCoverageFrom: [
    '<rootDir>/src/app-data/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/app-globals/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/cli/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/compiler/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/declarations/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/dev-server/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/hydrate/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/internal/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/mock-doc/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/runtime/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/screenshot/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/sys/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/testing/**/*.{js,jsx,ts,tsx}',
    '<rootDir>/src/utils/**/*.{js,jsx,ts,tsx}',
  ],
```
runs for me and doesn't seem to slow tests down too much. Are you sure you weren't accidentally picking up `node_modules`?

Reply:
🤔 Could have been! Didn't even think about that.

Reply:
Added that chunk in 526763c

Reply:
Yeah that does work much better. Thank you.